internal/store/build_reason.go

```diff
@@ -0,0 +1,24 @@
+package store
+
+type BuildReason int
+
+const BuildReasonNone = BuildReason(0)
+
+const (
+	BuildReasonFlagFiles BuildReason = 1 << iota
```

Comment:
Maybe this could be slightly more self explanatory. Like: `BuildReasonFlagMountFiles`?

Reply:
done